config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml

```diff
@@ -0,0 +1,41 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-bazel-job
+      decorate: true
+      run_if_changed: 'images/bazel'
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        containers:
+          - image: quay.io/powercloud/all-in-one:0.2
+            volumeMounts:
+              - name: kubeconfig
+                mountPath: /etc/kubeconfig
+            command:
+              - /bin/bash
+            args:
+              - -c
+              - |
+                export PATH=$GOPATH/bin:$PATH
```

Comment:
https://github.com/ppc64le-cloud/test-infra/blob/master/images/all-in-one/Dockerfile#L113
has the `$GOPATH/bin` to the `PATH` variable. Please check if this is needed.
Also, the all-in-one image seems to have `GO111MODULE=on`.